click_odoo_contrib/backupdb.py

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Copyright 2018 ACSONE SA/NV (<http://acsone.eu>)
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+import json
+import os
+import shutil
+import tempfile
+import zipfile
+
+import click
+import click_odoo
+from _dbutils import db_exists
+from click_odoo import odoo
+
+
+def _dump_db(dbname, backup):
+    with tempfile.NamedTemporaryFile() as f:
+        cmd = ["pg_dump", "--no-owner", "--format=c", dbname, "--file=" + f.name]
+        odoo.tools.exec_pg_command(*cmd)
+        backup.addfile(f.name, "dump.sql")
+
+
+def _create_manifest(cr, dbname, backup):
+    with tempfile.NamedTemporaryFile() as f:
```

Comment:
@sbidoul I'm not really satisfied with this method since the file is created twice... (temp + in backup)